### ISSUE TITLE
[viewer] Replacing backticks with expression

### ISSUE THREAD

**Issue by dtempcyer** [pedohorse/lifeblood#69]
When changing a parameter to use an expression enclosed in backticks, such as `` `node.name` `` the parameter window crashes and the entire interface stops responding with the following error:

```code
imgui.core.ImGuiError: ImGui assertion error ((g.FrameCount == 0 || g.FrameCountEnded == g.FrameCount) && "Forgot to call Render() or EndFrame() at the end of the previous frame?") at imgui-cpp/imgui.cpp:7090
Traceback (most recent call last):
  File "/home/alex/Lib/houdini/lifeblood-dev/install/f5d82467e4903/lifeblood_viewer/nodeeditor.py", line 1168, in drawForeground
    imgui.new_frame()
  File "imgui/core.pyx", line 3901, in imgui.core.new_frame
```
Now we have to forcibly remove the backticks and then change the parameter to use the expression.

lifeblood build: v0.9.0
OS: Manjaro 23.1.3 Vulcan

**Comment by dtempcyer** [comment]
Still getting this error when I try to enable expression. For example, if the test node has the `into` parameter make enable expression.

lifeblood build: bef06ea5c0e19
OS: Manjaro 23.1.3 Vulcan

**Comment by permahorse** [comment]
hmmm, cannot seem to reproduce it. what are exact set of actions?
now i'm trying this:
- open viewer
- create any node (i create "set attribute")
- set "count" to 1
- in "name" parameter i set some backtick expression, like ```bla`node.name` ```
- right click on this parameter -> enable expression

result: i'm getting backtick-expandable string converted to a simple python expression, no errors

**Comment by permahorse** [comment]
understood, the bug is now with non-string expressions, 
should be fixed with b8baef85058df63a6e7b08c2dcae77541e157de1 now
